### ISSUE TITLE
Trying to install - app has no manifest.json (?)

### ISSUE THREAD

**Issue by lapineige** [YunoHost-Apps/gogs_ynh#7]
Hi,
When trying to install this app, I have a "No manifest.json file", for CLI as well as Web Interface.
But you have in the repository. Is it an issue of manifest formatting ?

Tested on Yunohost 2.4 / Debian 8

**Comment by M5oul** [comment]
You need to accept JavaScript execution on your web browser.

**Comment by M5oul** [comment]
You have to accept JavaScript execution on the web browser to retrieve the manifest.
Finally, it seems to be a problem on YunoHost instance.

**Comment by lapineige** [comment]
Ok, I noticed that a XHR was blocked from raw.githubusercontent.com was blocked. It worked :)
Thanks a lot, blazing-fast answer :)

**Comment by opi** [comment]
@lapineige at first, did you have the same error with CLI ?

**Comment by lapineige** [comment]
Ok, finally I've got an Internal Server Error.

So trying with CLI, it does work, but I have a 404 Error on the web page.
I did it in verbose mode. Here are the last lines of the logs (I can provide the full version if needed)
>  sudo cp ../conf/nginx.conf /etc/nginx/conf.d/MYDOMAIN.TLD.d/gogs.conf
 '[' No = Yes ']'
 sudo systemctl restart rsyslog.service
 sudo systemctl reload nginx.service
Warning: Job for nginx.service failed. See 'systemctl status nginx.service' and 'journalctl -xn' for details.
 true
 sudo systemctl restart gogs.service
Success! The SSOwat configuration has been generated
Success! Installation complete

edit: Seeing nginx logs, seems like this error came from something else, not related to gogs.
edit: solved.
